node/service/src/chain_spec/moonshadow.rs

```diff
@@ -44,9 +44,9 @@ pub type ChainSpec = sc_service::GenericChainSpec<GenesisConfig, Extensions>;
 /// Generate a chain spec for use with the development service.
 pub fn development_chain_spec(mnemonic: Option<String>, num_accounts: Option<u32>) -> ChainSpec {
 	// Default mnemonic if none was provided
-	let parent_mnemonic = mnemonic.unwrap_or(
-		"bottom drive obey lake curtain smoke basket hold race lonely fit walk".to_string(),
-	);
+	let parent_mnemonic = mnemonic.unwrap_or_else(|| {
```

Comment:
I wonder if this pattern tends to increase our WASM size...

Reply:
Thats a good question. We could build a simple binary that does this and compile it to wasm32-unknown-unknown. Then do it the other way and see. That would probably be a good exercise in rust->wasm fundamentals